website/core/EjectBanner.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
I *think* this is appropriate as its own component? It'd be nice to use it in other places in the docs where it's not a whole-page affair.